examples/pipelines/checkov-project/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+nameSuffix: -checkov-project
+components:
+  - pkg:helm/dracon-oss-components/base
+  - pkg:helm/dracon-oss-components/git-clone
+  - components/producers/checkov
```

Comment:
this is ok when creating the component locally, however it will cause issues when the PR is merged and a release is created. it should be fixed but just before merging